graph/vertex/AttributeVertex.java

```diff
@@ -37,7 +37,7 @@
      *
      * @return the {@code ValueType} of this {@code Attribute}
      */
-    Encoding.ValueType valueType();
+    Encoding.ValueType<VALUE> valueType();
```

Comment:
propagate the new generic through the `AttributeVertex`, which makes the value type more useful